js/.husky/.gitignore

```diff
@@ -1 +0,0 @@
-_
```

Comment:
We no longer need this? I could not find any reference to it in the migration guide.